yarn.lock

```diff
@@ -13887,10 +13887,10 @@ merge@2.1.0:
   resolved "https://registry.npmjs.org/merge/-/merge-2.1.0.tgz"
   integrity sha512-TcuhVDV+e6X457MQAm7xIb19rWhZuEDEho7RrwxMpQ/3GhD5sDlnP188gjQQuweXHy9igdke5oUtVOXX1X8Sxg==
 
-merkletreejs@^0.2.31:
```

Comment:
wondering if there was any reason this was pinned to 0.2?

Reply:
Might have been a rec for a previous version?